resources/views/expandable.blade.php

```diff
@@ -8,6 +8,7 @@
     'expanded' => null,
     'placeholder' => null,
     'placeholderCount' => 1,
+    'showMore' => false,
```

Comment:
shouldn't this be `null` if it's supposed to be a slot? 🤔

Reply:
Yeah it should be indeed, will change